packages/lesswrong/components/ultraFeed/UltraFeedSpotlightItem.tsx

```diff
@@ -0,0 +1,645 @@
+import classNames from 'classnames';
+import React, { CSSProperties, useCallback, useRef, useEffect, useState } from 'react';
+import { userGetProfileUrlFromSlug } from '../../lib/collections/users/helpers';
+import { Link } from '../../lib/reactRouterWrapper';
+import { SECTION_WIDTH } from '../common/SingleColumnSection';
+import { getSpotlightUrl } from '../../lib/collections/spotlights/helpers';
+import { useStyles, defineStyles } from '../hooks/useStyles';
+import { descriptionStyles, getSpotlightDisplayTitle } from '../spotlights/SpotlightItem';
+import { useUltraFeedObserver } from './UltraFeedObserver';
+import { AnalyticsContext } from '@/lib/analyticsEvents';
+import FeedContentBody from "./FeedContentBody";
+import CloudinaryImage2 from "../common/CloudinaryImage2";
+import { useDialog } from '../common/withDialog';
+import UltraFeedPostDialog from './UltraFeedPostDialog';
+import UltraFeedItemFooter from './UltraFeedItemFooter';
+import ForumIcon from '../common/ForumIcon';
+import LWTooltip from '../common/LWTooltip';
+import { SHOW_ALL_BREAKPOINT_VALUE } from './ultraFeedSettingsTypes';
+import { isRegularClick } from '../posts/TableOfContents/TableOfContentsList';
+
+const SIDE_MARGIN = 150;
+
+const buildFadeMask = (breakpoints: string[]) => {
+  const mask = `linear-gradient(to right, ${breakpoints.join(",")})`;
+  return {mask, "-webkit-mask-image": mask};
+}
+
+const buildVerticalFadeMask = (breakpoints: string[]) => {
+  const mask = `linear-gradient(to bottom, ${breakpoints.join(",")})`;
+  return {mask, "-webkit-mask-image": mask};
```

Comment:
I think these don't need the prefixed version (https://caniuse.com/?search=mask-image)